tests/upgrade/basic/task.yaml

```diff
@@ -3,8 +3,11 @@ systems: [-ubuntu-core-16-64]
 restore: |
     rm -f /var/tmp/myevil.txt
 execute: |
+    # pull in apt helpers
```

Comment:
I'd suggest dropping these comments (here and in the other instance above). The line below feels super clear.